### PR TITLE
Add new endline survey questions

### PR DESCRIPTION
**Pull request by erikh360**
Changes in blue here https://www.figma.com/file/ZByDr14YBEKfxcEgKHGj2K/YAL-Baseline-Survey?type=whiteboard&node-id=824-7858&t=uOojkTdA84nfywww-4

### CODE REVIEW

File: yal/question_sets/endline/platform_review_endline.py [hunk 1]
```diff
@@ -117,6 +118,105 @@
...
+                    [
+                        "*When you visited the clinic or other health facility, what "
+                        "was the outcome? (If you had different experiences, please "
+                        "pick the response that was true most of the time).*"
+                    ]
+                ),
+                "options": [
+                    ("got_help", "I got help"),
+                    ("no_diagnosis", "Visited, no diagnosis"),
+                    ("no_help", "Didn`t get help"),
```
Comment (Hlamallama) [inline yal/question_sets/endline/platform_review_endline.py:196]: Didn't looks funny

File: yal/question_sets/endline/platform_review_endline.py [hunk 2]
```diff
@@ -117,6 +118,105 @@
...
+                        "*Was there a reason you didn’t go to the clinic or other "
+                        "health facility?*"
+                    ]
+                ),
+                "options": [
+                    ("where", "Didn`t know where"),
+                    ("time", "Didn`t have time"),
+                    ("money", "Didn`t have money "),
+                    ("judgement", "Fear of judgement"),
+                    ("bas_service", "Fear of bad service"),
```
Comment (Hlamallama) [inline yal/question_sets/endline/platform_review_endline.py:176]: bas_service

File: yal/question_sets/endline/platform_review_endline.py [hunk 3]
```diff
@@ -117,6 +118,105 @@
...
+                        "*When you visited the clinic or other health facility, what "
+                        "was the outcome? (If you had different experiences, please "
+                        "pick the response that was true most of the time).*"
+                    ]
+                ),
+                "options": [
+                    ("got_help", "I got help"),
+                    ("no_diagnosis", "Visited, no diagnosis"),
+                    ("no_help", "Didn`t get help"),
+                    ("dont_understand", "I don`t understand"),
```
Comment (Hlamallama) [inline yal/question_sets/endline/platform_review_endline.py:197]: Don't looks funny

File: yal/question_sets/endline/platform_review_endline.py [hunk 4]
```diff
@@ -117,6 +118,105 @@
...
+                ),
+                "options": [
+                    ("where", "Didn`t know where"),
+                    ("time", "Didn`t have time"),
+                    ("money", "Didn`t have money "),
+                    ("judgement", "Fear of judgement"),
+                    ("bas_service", "Fear of bad service"),
+                    ("elsewhere", "Got help elsewhere"),
+                    ("no_need", "No longer needed"),
+                    ("dont_understand", "I don`t understand"),
```
Comment (Hlamallama) [inline yal/question_sets/endline/platform_review_endline.py:179]: Don't

File: yal/question_sets/endline/platform_review_endline.py [hunk 5]
```diff
@@ -117,6 +118,105 @@
...
+            "endline_12_q6d_platform_review": {
+                "type": "list",
+                "text": "\n".join(
+                    [
+                        "*When you visited the clinic or other health facility, what "
+                        "was the outcome? (If you had different experiences, please "
+                        "pick the response that was true most of the time).*"
+                    ]
+                ),
+                "options": [
```
Comment (Hlamallama) [inline yal/question_sets/endline/platform_review_endline.py:193]: The options for 6d looks wrong

File: yal/question_sets/endline/platform_review_endline.py [hunk 6]
```diff
@@ -117,6 +118,105 @@
...
+                        "and mental health?*"
+                    ]
+                ),
+                "options": [
+                    ("a_lot", "A lot more likely"),
+                    ("little_more", "Little more likely"),
+                    ("no_change", "No change"),
+                    ("little_less", "Little less likely"),
+                    ("lot_less", "A lot less likely"),
+                    ("dont_understand", "I don`t understand"),
```
Comment (Hlamallama) [inline yal/question_sets/endline/platform_review_endline.py:217]: Don't looks funny

File: yal/question_sets/endline/platform_review_endline.py [hunk 7]
```diff
@@ -175,9 +275,8 @@
...
                 "text": "\n".join(
                     [
-                        "*Did you know you can request a callback from a "
-                        "LoveLife counsellor "
-                        "through the B-Wise WhatsApp chatbot?*"
+                        "*Did you know you can request a callback from a LoveLife "
+                        "counsellor through the BWise WhatsApp chatbot?*"
                     ]
                 ),
                 "options": [
```
Comment (Hlamallama) [inline yal/question_sets/endline/platform_review_endline.py:282]: missing No, didn’t need it option

**Review by Hlamallama — APPROVED** [review]

File: yal/question_sets/endline/sexual_health_literacy_endline.py [hunk 8]
```diff
@@ -400,17 +402,17 @@
...
                 ),
                 "options": [
                     ("yes_lot_more", "Yes, a lot more"),
                     ("yes_little_more", "Yes, a little more"),
-                    ("no", "No"),
+                    ("no", "No change"),
                     ("no_little_less", "No, a little less"),
                     ("no_lot_less", "No, a lot less"),
-                    ("dont_understand", "I don’t understand"),
+                    ("dont_understand", "I don`t understand"),
```
Comment (DevChima) [inline yal/question_sets/endline/sexual_health_literacy_endline.py:415]: Notice incorrect symbol used for apostrophe

File: yal/question_sets/endline/sexual_health_literacy_endline.py [hunk 9]
```diff
@@ -419,17 +421,17 @@
...
```
Comment (DevChima) [inline yal/question_sets/endline/sexual_health_literacy_endline.py:434]: Notice incorrect symbol used for apostrophe

**Pull request merged**